tests/e2e/builder/scripts/shared.ts

```diff
@@ -137,24 +142,51 @@ const updateConfigForTest = <BuilderType>(
   }
 };
 
+// generate a basic route.json for test (modern.js server required)
```

Comment:
why not generate the route.json in uni-builder

Reply:
done